io_scene_nif/nif_export.py

```diff
@@ -638,12 +639,14 @@ def execute(self):
                             ctrl.target = None
                 # oblivion
                 elif NifOp.props.game in ('OBLIVION', 'FALLOUT_3', 'CIVILIZATION_IV', 'ZOO_TYCOON_2', 'FREEDOM_FORCE_VS_THE_3RD_REICH'):
+                    b_armature = armature.get_armature()
+                    # TODO [animation] allow for object kf only
                     # create kf root header
                     kf_root = self.objecthelper.create_block("NiControllerSequence")
-                    if self.EXPORT_ANIMSEQUENCENAME:
-                        kf_root.name = self.EXPORT_ANIMSEQUENCENAME
-                    else:
-                        kf_root.name = filebase
+                    # if self.EXPORT_ANIMSEQUENCENAME:
```

Comment:
This is an override, wonder if we need this or have an alternative.
Think we would need to get this into the hands of users or test ourselves and see what happens

Reply:
In any case, it's not defined anywhere. Should it be exposed to the user in the export operator?

Reply:
Generally I would say this is a non-issue, unless nifs can have multiple roots.
Assuming one root only, the name of the nif root object directly corresponds to the name of the blender root object, regardless of its type (empty / armature).
For kf animations, if the name of the root NiControllerSequence matters at all, it can be set from the action name.

Reply:
Its not so much multiple roots, but different types of roots. Also to my knownledge they can have multiple roots, just we tend to ignore all of this stuff. But since we do, we can continue as is for now a look at it again.